legacy-redirects.js

```diff
@@ -17,6 +17,7 @@ const redirects = [
   { from: ['/stryker4s', '/stryker4s/index', '/stryker4s/quickstart'], to: '/docs/stryker4s/getting-started' },
   { from: ['/stryker/handbook', '/stryker4s/handbook', '/stryker-net/handbook'], to: '/docs/' },
   // Blogs
+  { from: ['/blog/tags/stryker', '/blog/tags/stryker/'], to: '/blog/tags/stryker-js' },
```

Comment:
I don't think this double slash is needed (and why the build fails)

Reply:
Thanks! This was the issue indeed